tests/vesting/FourYearVesting.js

```diff
@@ -776,7 +780,7 @@ contract("FourYearVesting", (accounts) => {
             await expectRevert(vesting.withdrawTokens(root, { from: a2 }), "unauthorized");
         });
 
-        it("shouldn't be possible to use governanceWithdrawVesting by anyone but owner", async () => {
+        it("shouldn't be possible to use governanceDierctWithdrawVesting by anyone but owner", async () => {
```

Comment:
typo

Reply:
Fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/c4c43c22e2dc30a936960e01e9a8acd6e2053616)